python/lesson03/solution/hello.py

```diff
@@ -33,9 +33,13 @@ def http_get(port, path, param, value):
     headers = {}
     tracer.inject(span, Format.HTTP_HEADERS, headers)
 
-    r = requests.get(url, params={param: value}, headers=headers)
-    assert r.status_code == 200
-    return r.text
+    try:
+        r = requests.get(url, params={param: value}, headers=headers)
+        assert r.status_code == 200
+        return r.text
+    except requests.ConnectionError as err:
```

Comment:
Should this be only catching this specific error type? If status code is not 200, then it's customary to also tag the span as erroneous.

Reply:
Noted.